rfc6962-bis.xml

```diff
@@ -1226,6 +1226,11 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
         </section>
       </section>
+      <section title="TLS Feature Extension">
+	<t>
+	  A CA MAY include the <xref target="tls_transinfo_extension">transparency_info</xref> extension number in a <xref target="RFC7633">TLS Feature Extension</xref> in root or intermediate certificates. This indicates that CT compliance is required for all certificates which are directly or indirectly issued by such certificates.
+	</t>
+      </section>
```

Comment:
LGTM, but I'd still like to see text explaining when/why to (or not to) include the "transparency_info" extension number in the TLS Feature extension in end-entity certificates.  Two possible options (and there may be others): 1) CAs SHOULD NOT do that; if they do, clients should ignore it; 2) CAs MAY do that; if they do, it means that clients MUST reject the cert if it is non-compliant with 6962-bis.